client/components/email-verification/email-unverified-notice.jsx

```diff
@@ -26,7 +25,6 @@ export default class EmailUnverifiedNotice extends React.Component {
 		this.updateVerificationState = this.updateVerificationState.bind( this );
 		this.handleDismiss = this.handleDismiss.bind( this );
 		this.handleSendVerificationEmail = this.handleSendVerificationEmail.bind( this );
-		this.handleChangeEmail = this.handleChangeEmail.bind( this );
 
 		this.state = {
 			needsVerification: userUtils.needsVerificationForSite( sites.getSelectedSite() ),
```

Comment:
This is probably out of scope, but keep in mind that SelectedSite.getSelectedSite() is deprecated. See example in https://github.com/Automattic/wp-calypso/issues/8726